docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md

```diff
@@ -0,0 +1,84 @@
+# Gas fee payment in WBTC using ERC-2771
+
+In this example, we will show how WBTC can be used for gas fee payments using the [ERC-2771 standard](https://eips.ethereum.org/EIPS/eip-2771) on the BOB testnet. This enables users to transact without the necessity to own ether.
+
+:::info Example Code
+
+The source code for this demo can be found in [this repository](https://github.com/bob-collective/demo-meta-transactions-transfer/).
+
+:::
+
+![preview](preview.png)
+
+## Local development
```

Comment:
I guess local setup can be useful as well, but didn't we want to host the page ourselves as well?

Reply:
Yes, we need to host these demos.